test/mysql_handler/test

```diff
@@ -1,20 +1,55 @@
 #!/bin/bash -ex
 
 local_test=false
-while getopts :l opt; do
+while getopts ":lb" opt; do
     case $opt in
         l) local_test=true ;;
+				b) benchmark=true ;;
        \?) echo "Unknown option -$OPTARG"; exit 1;;
     esac
 done
 
 docker_args=""
 if $local_test; then
-  docker_args="-v $(cd ../..; pwd):/go/src/github.com/cyberark/secretless-broker"
+  docker_args="-v $(cd ../..; pwd):/secretless"
 fi
 
-docker-compose run \
-  --rm \
-  --no-deps \
-  $docker_args \
-  test
+if $benchmark; then
+  # first set MYSQL_ADDRESS and SECRETLESS_ADDRESS
+  mysql_cid=$(docker-compose ps -q mysql)
+  secretless_cid=$(docker-compose ps -q secretless)
+  mysql_ip=$(docker inspect \
+    -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' $mysql_cid)
+  secretless_ip=$(docker inspect \
+    -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' $secretless_cid)
+  export MYSQL_ADDRESS=$mysql_ip:3306
+  export SECRETLESS_ADDRESS=$secretless_ip:15432
+
+  echo "++++++++++++++++++++++++++++++++++++++"
+  echo ""
+  echo "Running MySQL benchmarks ..."
+  echo ""
+  echo "++++++++++++++++++++++++++++++++++++++"
+
+	docker-compose run \
+		--rm \
+		--no-deps \
+		$docker_args \
+		test \
+		bash -c '
+		  echo "--- QUERYING MYSQL DIRECTLY ---" && \
+		  BENCH_ADDRESS=$MYSQL_ADDRESS go test -v -bench=. -test.benchtime=10s ./test/mysql_handler/bench_test.go | tee bench.old && \
+		   echo "--- QUERYING VIA SECRETLESS ---" && \
+		  BENCH_ADDRESS=$SECRETLESS_ADDRESS go test -v -bench=. -test.benchtime=10s ./test/mysql_handler/bench_test.go | tee bench.new && \
+		   echo "--- COMPARING BENCHMARKS ---" && \
+		  benchcmp bench.old bench.new
+		'
```

Comment:
This comment addresses lines 39 to 46.

Usage of single quotes prevents the interpolation of the envvars `MYSQL_ADDRESS` and `SECRETLESS_ADDRESS` from the script shell, instead they are interpolated from the docker-compose run shell of the test container, where their values are empty.

Perhaps use a heredoc:

```
bash << EOL
echo "--- QUERYING MYSQL DIRECTLY ---";
BENCH_ADDRESS=$MYSQL_ADDRESS go test -v -bench=. -test.benchtime=10s ./test/mysql_handler/bench_test.go | tee bench.old;
echo "--- QUERYING VIA SECRETLESS ---";
BENCH_ADDRESS=$SECRETLESS_ADDRESS go test -v -bench=. -test.benchtime=10s ./test/mysql_handler/bench_test.go | tee bench.new;
echo "--- COMPARING BENCHMARKS ---";
benchcmp bench.old bench.new
EOL
```